application/src/main/java/org/opentripplanner/street/model/vertex/VertexFactory.java

```diff
@@ -94,6 +94,14 @@ public ExitVertex exit(long nid, Coordinate coordinate, String exitName) {
     return addToGraph(new ExitVertex(coordinate.x, coordinate.y, nid, exitName));
   }
 
+  public StationEntranceVertex stationEntrance(
+    long nid,
+    Coordinate coordinate,
+    String entranceName
```

Comment:
Same entranceName vs code confusion here.

Reply:
Should entrance name be I18NString ?

Reply:
The name itself should probably be I18NString but this field is actually the code and I don't think translating it makes sense since it's typically something like `A` or `B`.